rpc/handler.go

```diff
@@ -73,6 +71,13 @@ type handler struct {
 	maxBatchConcurrency uint
 }
 
+// Custom bytes type which writes to JSON as is
+type JSONBytes []byte
```

Comment:
Native json.RawMessage can be used here?

Reply:
I will review tomorrow. Late here.

Reply:
Fixed